PowerEditor/src/WinControls/StaticDialog/RunDlg/RunDlg.rc

```diff
@@ -1,41 +1,37 @@
-/*
-This file is part of Notepad++ project
-Copyright (C)2021 Don HO <don.h@free.fr>
+// This file is part of Notepad++ project
+// Copyright (C)2021 Don HO <don.h@free.fr>
 
-This program is free software: you can redistribute it and/or modify
-it under the terms of the GNU General Public License as published by
-the Free Software Foundation, either version 3 of the License, or
-at your option any later version.
-
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-GNU General Public License for more details.
-
-You should have received a copy of the GNU General Public License
-along with this program.  If not, see <https://www.gnu.org/licenses/>.
-*/
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// at your option any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.

```

Comment:
@ozone10 
The reason of the following rule:
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md#comments

is while coding/debugging, devs can use `/**/` easily so there's imbrication situation. On the big bloc non-code zone, it can be allowed.

Reply:
ok